pkgs/os-specific/linux/kvmtool/default.nix

```
@@ -0,0 +1,49 @@
+{ stdenv, fetchgit, libbfd, zlib, libaio, glibc }:
+
+stdenv.mkDerivation {
+  name = "kvmtool";
+  version = "unstable-2020-05-19";
+
+  src = fetchgit {
+    url = "https://git.kernel.org/pub/scm/linux/kernel/git/will/kvmtool.git";
+    rev = "b4fc4f605fc66a0942e88f37b3a4b18671e32b0c";
+    sha256 = "0kyb3rm98wzwgjs06crcgb39547rr4px4vap8ivm7yqnwbphr67f";
+  };
+
+  buildInputs = [ libbfd zlib libaio glibc glibc.static];
```

Maybe add a comment why static glibc is needed:

```suggestion
  buildInputs = [
    libbfd
    zlib
    libaio
    glibc
    glibc.static # for guest init
  ];
```